### ISSUE TITLE
How to train a Large-from scratch Model?

### ISSUE THREAD

**Issue by SheepAndWolf** [pytorch/fairseq#3154]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?

**I have trained a pre-train Large model (Model-Y1) with my own unlabeled data, and finetuned it(Model-Y1) with my own labeled data. Now I get a finetuned model(Model-Y2)**
**I want to know how much Model-Y2 benifits from Model-Y1. So I try to train a model from scratch（without pretrain）.**

#### What have you tried?

I trained a pre-train model with a few data for one epoch.So I get a fake pre-trained model(Model-F1).Then I finetuned Model-F1 with my own data to get Model-F2.

the command I use:
`python train.py ${train_data} \
--no-pretrained-weights --feature-grad-mult 1 --freeze-finetune-updates 0 \
--save-dir ${save_dir} --valid-subset ${valid_subset} --w2v-path ${w2v_path} \
--distributed-world-size 16 --distributed-port 44675 --max-tokens 1920000 \
--max-update 320000 --warmup-steps 32000 --hold-steps 128000 --decay-steps 160000 \
--post-process letter --best-checkpoint-metric wer --num-workers 6 \
--sentence-avg --task audio_pretraining --arch wav2vec_ctc --labels ltr \
--optimizer adam --adam-betas '(0.9, 0.98)' --adam-eps 1e-08 --lr 3e-05 --lr-scheduler tri_stage \
--final-lr-scale 0.05 --final-dropout 0.0 --dropout 0.0 --activation-dropout 0.1 --criterion ctc \
--apply-mask --mask-selection static --mask-other 0 --mask-length 10 --mask-prob 0.5 \
--mask-channel-selection static --mask-channel-other 0 --mask-channel-length 64 --mask-channel-prob 0.256 \
--layerdrop 0.1 --zero-infinity --save-interval 1 --validate-interval 1 \
--attention-dropout 0.0 --seed 2337 --log-format json --log-interval 200 --ddp-backend no_c10d \
--skip-invalid-size-inputs-valid-test`

Infering by viterbi, Model-F2 get 40% WER in libri-speech test-other while Model-Y2 get 5.2%.I think something went wrong in Model-F2.
I don't know how to train a Large - from scratch Model, anyone could help?

#### What's your environment?
 - fairseq Version (0.10.1)
